### PR TITLE
Refactor iteration5

### PR DESCRIPTION
**Pull request by itsnameissteven**
## Is this a fix or a feature? ⬇️
This is fix
## What is the change? ⬇️
We deleted we edited the display card function for dynamic use

## What does it fix? ⬇️
The most current activity will be on top of the page. CSS properties were combined where appropriate or deleted.  HTML code should be the same just indented. 

## Where should the reviewer start? ⬇️
With the display card function. Make sure it is still working properly and saving the correct cards. 

## How should this be tested? ⬇️
Log new activities and refresh the page to make sure they still remain on the right side of the page.  All other features of the intention timer should be tested as well.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -218,12 +206,11 @@ input {
 }
 
 article {
+  display: flex;
   width: 90%;
   background-color: #46424D;
   margin-bottom: 25px;
-  border-radius: 5px;
   padding: 2%;
```
Comment (whitneywilkes) [inline styles.css:213]: Two padding props here on article - which should we go with?
lines 213 & 238 
Pushing with 0% for now.

**Review by whitneywilkes — COMMENTED** [review]
Does the CSS look correct?

File: styles.css (hunk 1 above)
Comment (itsnameissteven) [reply styles.css:213]: those might be addressing different things on the page

File: Activity.js [hunk 2]
```diff
@@ -2,34 +2,18 @@ class Activity {
...
-    timer.innerText = `${minutes.toString().padStart(2, '0')}:${seconds.toString().padStart(2 , '0')}`;
-  }, 1000);
+  startTimer() {
+    displayTime(this.minutes, this.seconds)
   };
 
   markComplete() {
-    startButton.innerText = "COMPLETE!";
-    removeClass(logButton);
+    this.completed = true
```
Comment (JeffShepherd) [inline Activity.js:16]: Updated to separate data model and DOM

File: main.js [hunk 3]
```diff
@@ -32,13 +23,12 @@ var startNewActivityForm = document.querySelector('.start-new-activity')
...
-window.addEventListener('load', displayLocalStorage)
-
+window.addEventListener('load', displayLocalStorage);
 logButton.addEventListener("click", displayCard);
 startActivityButton.addEventListener("click", startActivity);
-createNewActivityButton.addEventListener("click", window.location.reload)
-
+createNewActivityButton.addEventListener("click", function() {
+  window.location.reload()
+})
```
Comment (JeffShepherd) [inline main.js:31]: Works perfect and gets rid of the 'illegal invocation" console error we were getting.

File: main.js [hunk 4]
```diff
@@ -51,9 +41,8 @@ selectActivityButton.addEventListener("click", function(event) {
   button.firstElementChild.src = `./assets/${button.value.toLowerCase()}-active.svg`;
 });
 
-
 timerContainer.addEventListener("keydown", function(event) {
-  var invalidCharacters = ["e", "+", "-"];
+  var invalidCharacters = ["e", "+", "-", "."];
```
Comment (JeffShepherd) [inline main.js:45]: Works great and disallows decimal values

File: main.js [hunk 5]
```diff
@@ -126,55 +117,64 @@ function resetButtons() {
...
+      } else {
+        seconds--
+      }
+      fixTime(minutes, seconds)
+    }, 1000);
 }
+
+function fixTime(minutes, seconds) {
+  timer.innerText = `${minutes.toString().padStart(2, '0')}:${seconds.toString().padStart(2 , '0')}`
+}
```
Comment (JeffShepherd) [inline main.js:180]: Works as expected and removes the same reassignment line being used in multiple functions previously.

File: styles.css (hunk 1 above)
Comment (JeffShepherd) [reply styles.css:213]: fixed in subsequent push

**Review by JeffShepherd — APPROVED** [review]
All changes made while screen-sharing. Most recent push also pulled down and tested locally on another machine. Looks good.

**Pull request merged**